templates/install-wp-tests.sh

```diff
@@ -57,6 +64,22 @@ install_wp() {
 	else
 		if [ $WP_VERSION == 'latest' ]; then
 			local ARCHIVE_NAME='latest'
+		elif [[ $WP_VERSION =~ [0-9]+\.[0-9]+ ]]; then
+			# https serves multiple offers, whereas http serves single.
+			download https://api.wordpress.org/core/version-check/1.7/ /tmp/wp-latest.json
+			if [[ $WP_VERSION =~ [0-9]+\.[0-9]+\.[0] ]]; then
+				# version x.x.0 means the first release of the major version, so strip off the .0 and download version x.x
+				WP_VERSION=${WP_VERSION%??}
```

Comment:
Maybe the variable here should be renamed, since `LATEST_VERSION` doesn't make so much sense when fetching the initial major release.  Perhaps using a variable called something like`DOWNLOAD_VERSION` would be clearer?